liver_imaging_analysis/engine/engine.py

```diff
@@ -333,11 +435,13 @@ def per_epoch_callback(self, *args, **kwargs):
 
     def fit(
         self,
-        epochs = config.training["epochs"],
-        evaluate_epochs = 1,
-        batch_callback_epochs = None,
-        save_weight = False,
-        save_path = config.save["potential_checkpoint"],
+        summary_writer,
```

Comment:
These should have default values, it doesn't make sense to have them as mandatory arguments.